Dockerfile

```diff
@@ -186,6 +203,7 @@ ARG UID=1000
 ARG GID=10001
 RUN groupadd --gid $GID guacd
 RUN useradd --system --create-home --shell /sbin/nologin --uid $UID --gid $GID guacd
+COPY ./src/guacd-docker/krb5.conf /etc/krb5.conf
```

Comment:
I'm not sure we should provide this - maybe just document the instructions for it.